### ISSUE TITLE
Unlock fails after changing the language for terminal 

### ISSUE THREAD

**Issue by makuru-org** [sgillespie/nixos-yubikey-luks#8]
I changed the language / keymap from German to English and now i can't login anymore.

I saw this problem coming and that's why I added an key for us but that doesn't work because of the change to the seed.

Possible fixes, 

> force us keymap on boot

> Don't change the seed every boot / have an option to disable it

Additional add a big fat warning about it, in the user guide.

**Comment by sgillespie** [comment]
Yeah I think this would definitely be a problem. I also use a different layout, but has not caused a problem (probably because I'm using xkb).
